deps/checksums/curl

```diff
@@ -1,40 +1,38 @@
-LibCURL-9ea5c5d6f5b88615d9fe23379b7f951787b99fd3.tar.gz/md5/a1e927b166ffd98c83436d21588a5d19
-LibCURL-9ea5c5d6f5b88615d9fe23379b7f951787b99fd3.tar.gz/sha512/be87705fd4232f59a1a39aef6594ca2b34823ec411ae2e2ee1463836263f88599c6aadc9c0d9c635bae399761daba38046169d8cc5e02e7531db104037e7177c
```

Comment:
First two lines disappeared. I also don't know what they're needed for. @vtjnash?

Reply:
Looks like a download for a source build at a particular checkout instead of a version tag. Not sure what it would be from previously